ambry-server/src/main/java/com.github.ambry.server/StatsManager.java

```diff
@@ -134,7 +135,7 @@ void collectAndAggregate(StatsSnapshot aggregatedSnapshot, PartitionId partition
       try {
         long fetchAndAggregatePerStoreStartTimeMs = time.milliseconds();
         StatsSnapshot statsSnapshot = store.getStoreStats().getStatsSnapshot();
-        aggregate(aggregatedSnapshot, statsSnapshot);
+        StatsSnapshot.aggregate(aggregatedSnapshot, statsSnapshot);
```

Comment:
will you put up a follow up PR to remove the scheduler within StatsManager which aggregates and publishes the stats locally